src/exporter/trace/stdout/mod.rs

```diff
@@ -0,0 +1,49 @@
+use crate::exporter::trace;
+use std::any;
+use std::sync::Arc;
+
+#[derive(Debug)]
+/// Exporter
+pub struct Exporter {}
+
+#[derive(Debug)]
+/// Builder
+pub struct Builder {}
+
+impl Default for Builder {
+    /// Return the default Exporter Builder.
+    fn default() -> Self {
+        Builder {}
+    }
+}
+
+impl Builder {
+    /// Create a new exporter from the builder
+    pub fn init(self) -> Exporter {
+        Exporter {}
+    }
+}
+
+impl Exporter {
+    /// Create a new exporter builder.
+    pub fn builder() -> Builder {
```

Comment:
Why can `Exporter`s create `Builder`s? That direction seems counter-intuitive to me.

Reply:
Indeed, I get the skeleton from jaeger exporter, but the usage is a little different, cleaning up as you suggested, thanks :)